### DISCUSSION TITLE
Is there a way to change an alarm from High to low value?

### DISCUSSION THREAD

**Discussion by BL-Shopwork** [Serial-Studio/Serial-Studio#203] [Q&A] [answered]
Love the software BTW, its really well done.

Is there a way that the alarm feature can be expanded or I can expand it? Specifically if there was a High and Low alarm setting and if possible the "LED" changes colour when an alarm event happens?

**Comment by alex-spataru** [comment]
Hi, for the moment nope. I will add this feature for the next release. I think that adding a "LED Low Value" and a "LED High Value" for the dataset items will suffice. I am currently revamping Serial Studio, and made some changes that simplify a lot the implementation of the project editor by using Model/View programming.

Check [this](https://github.com/Serial-Studio/Serial-Studio/discussions/184#discussioncomment-10656085) comment for a screenshot of how the new interface is beginning to look :)

I'll let you know when a build that fixes your issue is available.

**Comment by alex-spataru** [comment, marked as answer]
Just a heads-up: starting from version 3.0.0, the latest stable releases include a new field in the project editor called 'LED High (On) Value.' When the real-time value meets or exceeds this threshold, the LED will turn on. 

If the real-time value meets or exceeds the alarm threshold, the LED (and its label) shall turn red. The alarm-level behavior now applies to the following widgets:

- Dataset grid
- Gauge
- Bar
- LED
